docs/ref/models/database-functions.txt

```diff
@@ -149,6 +149,41 @@ will result in a database error.
     The PostgreSQL behavior can be emulated using ``Coalesce`` if you know
     a sensible maximum value to provide as a default.
 
+``NullIf``
+----------
+
+.. class:: NullIf(expression1, expression2)
+
+.. versionadded:: 2.1
+
+Accepts two arguments, and returns ``None`` if they are equal, otherwise
+returns ``expression1``.
+
+.. admonition:: Caveats when using Oracle
+
+    Due to convention, this function returns the empty string instead of
+    ``None`` when the expressions are of type
+    :class:`~django.db.models.CharField`. See the :ref:`Oracle notes
+    <oracle-notes>` for more details.
+
+    It is not possible to pass ``Value(None)`` to ``expression1`` as Oracle
```

Comment:
Passing `Value(None)`` to ``expression1`` is prohibited on Oracle since Oracle doesn't accept...